### DISCUSSION TITLE
Using TinyPNG Image Compression From MacOS Finder Contextual Menu | devDosvid blog

### DISCUSSION THREAD

**Discussion by giscus** [vasylenko/devdosvid.blog#86] [General]
# Using TinyPNG Image Compression From MacOS Finder Contextual Menu | devDosvid blog

How to add TinyPNG image compression to your macOS Finder contextual menu

https://devdosvid.blog/2021/02/14/using-tinypng-image-compression-from-macos-finder-contextual-menu/

**Comment by meghantosh** [comment]
Thank you!  This was just what I needed -- it AMAZES me that graphics programs don't have this built in yet. THANK YOU THANK YOU

**Reply by vasylenko** [reply]
@meghantosh I am glad it was useful to you! ❤️
